python/friesian/example/wnd/README.md

```diff
@@ -5,11 +5,14 @@ This example demonstrates how to use BigDL Friesian to preprocess the
 ## Prepare the environment
 We recommend you to use [Anaconda](https://www.anaconda.com/distribution/#linux) to prepare the environments, especially if you want to run on a yarn cluster (yarn-client mode only).
 ```
-conda create -n zoo python=3.7  # "zoo" is the conda environment name, you can use any name you like.
-conda activate zoo
-pip install --pre --upgrade analytics-zoo
+conda create -n bigdl python=3.7  # "bigdl" is the conda environment name, you can use any name you like.
+conda activate bigdl
+pip install --pre --upgrade bigdl-friesian
+```
+You also need to install tensorflow in this example.
+```
+conda install tensorflow==2.6.0
```

Comment:
just put it together with the previous section like this:
https://github.com/intel-analytics/BigDL/tree/branch-2.0/python/friesian/example/dien#prepare-the-environment
Use pip install tensorflow